packages/graphics/src/Graphics.ts

```diff
@@ -763,33 +837,27 @@ export class Graphics extends Container
      * @param {number[]|PIXI.Point[]|PIXI.Polygon} path - The path data used to construct the polygon.
      * @return {PIXI.Graphics} This Graphics object. Good for chaining method calls
      */
-    drawPolygon(path)
+    drawPolygon(path: Array<number> | Array<Point> | Polygon): this
     {
         // prevents an argument assignment deopt
         // see section 3.1: https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#3-managing-arguments
-        let points = path;
+        let points: Array<number> | Array<Point>;
 
         let closeStroke = true;// !!this._fillStyle;
 
         // check if data has points..
-        if (points.points)
+        if ((path as Polygon).points)
         {
-            closeStroke = points.closeStroke;
-            points = points.points;
+            closeStroke = (path as Polygon).closeStroke;
+            points = (path as Polygon).points;
         }
-
-        if (!Array.isArray(points))
+        else
         {
```

Comment:
Why remove support for `...args:number[]` here? These TS conversions shouldn't remove functionality.

Reply:
Same problem in Poligon package now. We doesn't support arrays in constructor now.
Can you write 2 annotation with ... and [], because i wasn't resolve this.

Reply:
Oke, i restore it.
Tests will come later.